### PR TITLE
Add Optional Padding For Dialog

### PR DESCRIPTION
**Pull request by dogukankeskinoglu**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Description
ScrollView of Dialog has default horizontal padding(16 dp) but custom dialog shouldn't have padding other than zero. With this feature, we can set padding for any dialog

## Checklist:

- [x] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Review by turkergoksu — APPROVED** [review]

File: libraries/dialogs/README.md [hunk 1]
```diff
@@ -174,6 +176,8 @@ CustomDialog is a dialog that has a fixed header and its content can take a cust
 | Field | Type | Description | Default Value |
 | ------------- |-------------|-------------| ------------- |
 | `view` | View | The custom view that will create the body of the dialog. | null |
+| `horizontalPadding` | Float | Horizontal padding will be applied. | 0dp |
```
Comment (bilgehankalkan) [inline libraries/dialogs/README.md:179]: Why default is different than the above?

File: buildSrc/src/main/kotlin/ComponentVersions.kt [hunk 2]
```diff
@@ -5,7 +5,7 @@ object ComponentVersions {
     const val ratingBarVersion = "1.0.2"
     const val imageSliderVersion = "1.0.8"
     const val phoneNumberVersion = "1.0.2"
-    const val dialogsVersion = "1.2.9"
+    const val dialogsVersion = "1.3.0"
```
Comment (bilgehankalkan) [inline buildSrc/src/main/kotlin/ComponentVersions.kt:8]: Don't forget to update version info on README.md#4

**Review by bilgehankalkan — APPROVED** [review]

**Review by bsobe — APPROVED** [review]

**Review by cifo19 — APPROVED** [review]

File: libraries/dialogs/README.md (hunk 1 above)
Comment (dogukankeskinoglu) [reply libraries/dialogs/README.md:179]: Main purpose of custom dialog is set view from client. So if we want to use custom dialog, we have to set horizontalPadding all the time. I think we should handle padding from custom view

File: buildSrc/src/main/kotlin/ComponentVersions.kt (hunk 2 above)
Comment (dogukankeskinoglu) [reply buildSrc/src/main/kotlin/ComponentVersions.kt:8]: Done https://github.com/Trendyol/android-ui-components/pull/93/commits/46670bfd2e8113bfb2b4723bd0e71e49bffd893e

**Pull request merged**